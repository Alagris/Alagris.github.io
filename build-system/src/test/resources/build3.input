:eval function1 '01'
:eval function1 ''
:eval function2 'this function has no type, hence it defaults to using .* as alphabet'
:eval mirror 'reflect a input'
:eval mirror 'reflect b input'
:eval mirror 'reflect z input'
:eval auto_exponentials 'abckabc'
:eval auto_exponentials 'redk'
:eval auto_exponentials 'kredredred'
:eval referenced_across_files 'aaaaaa'
:eval multiple_types 'aaa'
:eval used_ahead_of_definition 'terabckabckre'
:eval used_ahead_of_definition 'terabckkre'
:eval used_ahead_of_definition 'terkredredredkre'
:eval sample3 'aaaaaaa'
:eval from_ostia 'aaaaaaa0'
x = !!function1 '1'
:eval x '011'