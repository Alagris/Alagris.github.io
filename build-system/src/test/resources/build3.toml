
name="Test project 3"
authors = ["Willy Wonka"]
version = "1.0.0"

[[source]]
path = "sample.mealy"
[[source]]
path = "sample2.mealy"
[[source]]
path = "sample3.ostia"
[[source]]
path = "sample3.mealy"