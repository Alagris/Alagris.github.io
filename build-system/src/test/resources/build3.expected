'011'
'10'
''
'a'
'b'
'z'
''
''
''
''
''
''
''
''
'qqqqqqqb'
'qqqqqqqb1'
